admin/builders/osx64_build.sh

```
@@ -85,24 +90,24 @@ echo "The Builder $0 is building the Percolator packages with src=${src_dir} an\
 d build=${build_dir} for user" `whoami`
 $package_manager install $other_packages
 $package_manager install $boost_install_options
+
+if [[ -f /usr/local/bin/brew || -f /opt/homebrew/bin/brew ]]
+  then
+  export HOMEBREW_PREFIX="$(brew --prefix)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)